modules/tracing/mule-tracer-configuration-impl/src/main/java/org/mule/runtime/tracing/level/impl/config/FileTracingLevelConfiguration.java

```diff
@@ -51,41 +52,53 @@
  */
 public class FileTracingLevelConfiguration extends FileConfiguration implements TracingLevelConfiguration, Disposable {
 
-  private final String CONFIGURATION_FILE_PATH =
-      getProperty(MULE_OPEN_TELEMETRY_TRACING_CONFIGURATION_FILE_PATH,
-                  getConfFolder() + FileSystems.getDefault().getSeparator() + getPropertiesFileName());
-
-  private final MuleContext muleContext;
-
   private static final String CONFIGURATION_FILE_NAME = "tracing-level.conf";
   private static final String LEVEL_PROPERTY_NAME = "mule.openTelemetry.tracer.level";
   private static final String OVERRIDES_PROPERTY_NAME = "mule.openTelemetry.tracer.levelOverrides";
+
   private static final Logger LOGGER = getLogger(FileTracingLevelConfiguration.class);
+  private final String CONFIGURATION_FILE_PATH =
+      getProperty(MULE_OPEN_TELEMETRY_TRACING_CONFIGURATION_FILE_PATH,
+                  getConfFolder() + FileSystems.getDefault().getSeparator() + getPropertiesFileName());
+
   private final HashMap<String, TracingLevel> tracingLevelOverrides = new HashMap<>();
-  private boolean tracingConfigurationFileWatcherInitialised;
+  private final List<Runnable> onConfigurationChangeRunnables = synchronizedList(new ArrayList<>());
+  private final LazyValue<TracingLevel> tracingLevelInitializer = new LazyValue<>(() -> {
+    initialise();
+    return null;
+  });
   private TracingLevel tracingLevel = null;
-  private JsonNode configuration;
 
-  private List<Runnable> onConfigurationChangeRunnables = synchronizedList(new ArrayList<>());
+  private final MuleContext muleContext;
+  private JsonNode configuration;
   private URL configurationUrl;
-  private TracingConfigurationFileWatcher tracingConfigurationFileWatcher;
   private ConfigurationPropertiesResolver propertyResolver;
-  private boolean initialised;
+  private TracingConfigurationFileWatcher tracingConfigurationFileWatcher;
```

Comment:
The tracing level configuration watcher should not be here either. this should be decoupled to the other file as in the previous configuration.